src/service/ComposeService.ts

```diff
@@ -293,6 +297,66 @@ export class ComposeService {
                             ...n.compose,
                         }),
                     );
+                    if (n.httpsProxy) {
+                        if (presetData.httpsProxies?.length) {
+                            if (presetData.nodes?.length && presetData.nodes[0].host) {
+                                const httpsProxyPreset = presetData.httpsProxies[0];
+                                httpsProxyPreset.excludeDockerService = false;
+                                const restGatewayDomains = resolveHttpsProxyDomains(
+                                    presetData.nodes[0].host,
+                                    `http://${n.name}:${internalPort}`,
+                                );
+                                if (httpsProxyPreset.domains) {
+                                    // domains already defined
+                                    if (httpsProxyPreset.domains.indexOf(n.name) < 0) {
+                                        // if rest-gateway not in the domains then add it
+                                        httpsProxyPreset.domains += ',' + restGatewayDomains;
+                                    }
+                                } else {
+                                    httpsProxyPreset.domains = restGatewayDomains;
+                                }
+                                if (typeof n.httpsProxy === 'number' && httpsProxyPreset.openPort === 3001) {
```

Comment:
is  the "&& httpsProxyPreset.openPort === 3001" ok?

Reply:
Tried to check if the `openPort` in the `httpsProxies`is not altered and still stays as default then override it with the `httpsProxy` number value, makes sense?